dev-python/pylatex/pylatex-1.3.3.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="PyLaTeX"
+MY_P="${MY_PN}-${PV}"
+
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+inherit distutils-r1 eutils
+
+DESCRIPTION="A Python library for creating LaTeX files and snippets"
+HOMEPAGE="https://github.com/JelteF/PyLaTeX"
+SRC_URI="https://github.com/JelteF/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples"
+
+RDEPEND="
+	dev-python/ordered-set[${PYTHON_USEDEP}]
+"
+
+distutils_enable_sphinx docs/source dev-python/sphinx_rtd_theme
+distutils_enable_tests nose
```

Comment:
Put these below `BDEPEND`.

Reply:
Is this is because one wants all XDEPEND variables defined at the same place in the ebuild? If not can you explain why this change?

I have locally made all the changes you requested and am testing the new ebuilds according to https://wiki.gentoo.org/wiki/Chroot_for_package_testing. Will rebase when testing is finished.

Thank you for the review!

Reply:
There are warnings in building the documentation which is in fact incomplete. I filed an issue upstream (https://github.com/JelteF/PyLaTeX/issues/285) and will modify this PR when it's resolved.

Reply:
> Is this is because one wants all XDEPEND variables defined at the same place in the ebuild? If not can you explain why this change?
> 

It's because I thought `d_e_t` appends its value to already defined RDEPEND and BDEPEND variables, so you'd want to inherit it after defining RDEPEND or BDEPEND. However I checked https://dev.gentoo.org/~mgorny/python-guide/distutils.html#using-distutils-enable-tests and remembered it wrong, it only needs to be called after RDEPEND, then do the BDEPEND like you had. Sorry.